.npm/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@avenga/couper",
+  "version": "0.0.1",
+  "description": "A lightweight API gateway designed to support API-driven Web projects",
+  "keywords": ["couper", "API", "gateway"],
+  "license": "MIT",
+  "homepage": "https://couper.io/",
+  "bin": {
+    "couper": "couper"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/avenga/couper"
+  },
+  "scripts": {
+    "start":   "couper run",
+    "couper":  "couper",
+    "verify":  "couper verify",
+    "help":    "couper help",
+    "version": "couper version",
+    "postinstall": "node install.js"
+  },
+  "main": "couper-binary.js",
+  "dependencies": {
```

Comment:
how can we make sure that our "install dependencies" don't leak into the including project's bundle? is `devDependencies` good enough?

Reply:
`devDependencies` won't work here, those packages are required during installation.

Production setups will presumably be based on containers, right? Having an os-specifc `couper` binary inside a `couper` container does not make sense. Therefore, `couper` itself should become a `devDependency`:

    $ npm install -save-dev @avenga/couper